R/FineTuningAndPrediction.R

```diff
@@ -197,22 +204,24 @@ trainModel <- function(pretrainedModelFolder,
                        personSequenceFolder,
                        fineTunedModelFolder,
                        trainingSettings,
-                       modelType) {
+                       predictionHead) {
   dir.create(fineTunedModelFolder)
   modelSettings <- yaml::read_yaml(file.path(pretrainedModelFolder, "model.yaml"))
+  batchSize <- trainingSettings$batchSize
+  trainingSettings$batch_size <- NULL
   trainModelSettings <- list(
     system = list(
       sequence_data_folder = personSequenceFolder,
-      output_folder= fineTunedModelFolder,
+      output_folder = fineTunedModelFolder,
       pretrained_model_folder = pretrainedModelFolder,
-      batch_size = as.integer(32),
-      checkpoint_every = as.integer(10)
+      batch_size = batchSize,
+      checkpoint_every = as.integer(1)
```

Comment:
In my experience deep learning models converge quickly on our data so during testing I saved checkpoints every epoch. I should move this to a user-set variable though and add an option so save checkpoint only if validation score improves.